src/components/layout/styles.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components'
+
+export const Container = styled.div`
```

Comment:
maybe?
```suggestion
export const Container = styled.main`
```
and do not wrap on the `content/index.tsx`

Reply:
Container can be repeated multiple times on the view. For instance in Header of Footer.
The content is unique and uses `main` element and the container again. 🙂

Reply:
To illustrate this is the structure in the main Layout. They all use `Container` to restrict the width but only `Content` is the place where the main content will be rendered.
<img width="253" alt="Screenshot 2020-11-02 at 19 59 29" src="https://user-images.githubusercontent.com/3129012/97907729-ff69ba00-1d45-11eb-8cb4-2a483f00d12a.png">

Reply:
This can be solved by keeping the Container as `div` and using the `as="main"` for that one use case.  There aren't supposed to be multiple `main` elements therefore the use of `as` should be quite sufficient. Not sure about further use of the `Content` component but if it's only purpose is to be a main container you can completely ditch it and replace that one usage with `Container as="main"`

Reply:
Seems like a good idea.